src/workers/notifications/postAdded.ts

```diff
@@ -66,6 +74,23 @@ const worker: NotificationWorker = {
             } as NotificationPostContext & Partial<NotificationDoneByContext>,
           }),
         );
+
+        const posts = await con.getRepository(SharePost).countBy({
```

Comment:
Idea for optimization: What if we add a migration to add `UserAction` for everyone who already published a post and then the only thing we need to do here is to check the `UserAction` table like below. This will remove a query to count the number of articles every time a user publishes. Wdyt?

Reply:
Good point! Much easier to check a single row than a potential many! Let's do that 🚢